frontend/Makefile

```
@@ -6,89 +6,42 @@ OC ?= oc
-
-dev-teardown: ## Remove the dev plugin deployment from the cluster
-	@$(OC) get console.operator.openshift.io cluster -o json | \
-		jq '.spec.plugins |= (. // [] | map(select(. != "$(PLUGIN_NAME)")))' | \
-		$(OC) apply -f -
-	@$(OC) delete -f deploy/consoleplugin.yaml --ignore-not-found
-	@$(OC) delete -f deploy/dev-deployment.yaml --ignore-not-found
-	@$(OC) delete namespace $(PLUGIN_NAMESPACE) --ignore-not-found
-	@$(OC) rollout restart deployment/console -n openshift-console
-	@$(OC) rollout status deployment/console -n openshift-console --timeout=120s
+.PHONY: help build deploy teardown test
 
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)